app/src/main/res/values/colors.xml

```diff
@@ -8,5 +8,5 @@
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
     <color name="ymAccentBlue">#FF03DAC5</color>
-    <color name="ym_icon_color">#FFFFFF</color>
```

Comment:
Why have we commented the icon color?

Reply:
Not required.. this is an example how once can override